### DISCUSSION TITLE
File name patterns for release notes

### DISCUSSION THREAD

**Discussion by fjakop** [renovatebot/renovate#12446] [Request Help] [answered]
Hello folks,

thanks for this great project, which we are using self hosted on our k8s platform. 

We are wondering whether there is some documentation about the fetched file names for the release notes, because our projects are using "CHANGELOG.adoc" instead of "CHANGELOG.md" and renovate does not fetch release notes for these projects.
We also found no config parameter or docs about this.

Can someone explain how this is intended to work?

Kind regards
Frank

**Comment by rarkins** [comment, marked as answer]
Renovate uses https://github.com/shinnn/changelog-filename-regex

BTW even if Renovate detected the file name, it might not be able to parse asciidoc headings/releases unless it's by chance, so would likely need code changes for that too

**Reply by fjakop** [reply]
Thanks a lot @rarkins , then we will try markdown first.
